src/localize.js

```diff
@@ -125,20 +125,58 @@ const strategies = {
   },
 };
 
+// Reference for ratios https://www.w3.org/International/articles/article-text-size
+const expansionRatio = {
+  10: 2.5,
+  20: 1.9,
+  30: 1.7,
+  40: 1.5,
+  50: 1.5,
+  60: 1.6,
+  70: 1.6,
+  over70: 1.3,
+};
+
+const isAlpha = character => {
+  return /[a-zA-Z]/.test(character);
+};
```

Comment:
this is testing if the character is an ASCII character right? Should the be called `isASCII` ?

Reply:
This returns “True” if all characters in the string are alphabets, Otherwise, It returns “False”. ASCII = Alphanumeric, however here I target only alphabetic characters.